documentation/modules/vmware-prerequisites.adoc

```diff
@@ -14,6 +14,7 @@ The following prerequisites apply to VMware migrations:
 * You must create a VMware Virtual Disk Development Kit (VDDK) image.
 * You must obtain the SHA-1 fingerprint of the vCenter host.
 * If you are migrating more than 10 VMs from an ESXi host in the same migration plan, you must increase the NFC service memory of the host.
+* You must disable hibernation.
```

Comment:
@RichardHoch maybe we can be less strict here with something like:
Make sure there are no hibernated VMs. It is recommended to disable hiberation.

I just looked at t[he guidelines on how to disable hibernation in vsphere](https://docs.vmware.com/en/VMware-Horizon-7/7.13/published-desktops-applications/GUID-69E4CC65-7DD1-43B0-A522-8ABFEB9388EF.html) and users may want to avoid it when they read `When you make hibernation unavailable, hybrid sleep does not work. Users can lose data if a power loss occurs.`. It's not a strict prerequisite on our end but a recommendation so migrations won't fail on hibernated VMs

Reply:
@RichardHoch & @ahadas thanks for the clarity.
I am concerned and want to ensure we cover this correctly.
I would like to include an IMPORT admonition, something like:

> [!IMPORTANT]
>
> It is recommended to disable hibernation.
> When you disable hibernation, hybrid sleep will not work. If there is a power loss, it is possible you could lose data.

Richard can make it look prettier, but would you be happy with the general context?

Reply:
I phrased it somewhat differently, with fewer details -- @ahadas I'm open to including more information. Let me know what you think is best for our users.